coc.md

```diff
@@ -18,6 +18,20 @@ layout: page
 		<p>Some LGBTQ in Tech spaces may have additional rules in place, which will be made clearly available to participants. Participants are responsible for knowing and abiding by these rules.</p>
 
 		<div class="hr"></div>
+
+		<a name="conduct"></a>
+		<h2>Conduct</h2>
+		<div class="row">
+			<div class="medium-12 columns">
+				<ul>
+					<li> No <a href='#harassment'>harassment</a></li>
+					<li> No advocating for members to be banned, except in direct messages, or discussion with admins </li>
```

Comment:
> or discussion with the admins

Could this be "or _private_ discussion with the admins"? Otherwise a public conversation with an admin would be a permitted place to call for a ban.

Reply:
Good call!